COVID-19-electronic-health-system/Coronalytics
# mock data

fixes #10 

### Please provide enough information so that others can review your pull request:

Mock data generated and structured according to current data structure on blockstack. Rails API deployed on Heroku and mock data accessible at any time. 

### Explain the **details** for making this change. What existing problem does the pull request solve?

### Test plan (required)

To start this locally:
- Run ```rvm use Ruby-2.6.1``` to make sure you are using the correct version of Ruby
- Run ```bundle install``` to install all gems/dependancies required for Rails API
- Run ```rails db:create``` to create a local PostgresQL database
- Run ```rails db:migrate``` to create the tables/columns required for proper Rails API functionality
- Run ```rails db:seed``` to populate the columns with rows of "mock" data

To run server:
Run ```rails s```  to start the backend server on port 3000

Go to ```http://localhost:3000/users``` to see the data in json form.

### Final Checklist

- [x] Have you bumped the version in `package.json`?
    - Second decimal for major change, third decimal for minor change. This can go past 10 i.e. 1.0.9 !=> 1.1.0, 1.0.9 => 1.0.10
    - [Read here for more info](https://semver.org/)
- [x] Have you added any new tests necessary?
- [x] Is your PR rebased off the most current master?
- [x] Have you squashed all commits, or if you will merge will you squash all commits?
- [x] Did you use yarn, not npm?
- [x] Did you use Material-UI wherever possible?
- [x] Did you format according to Prettier?
- [x] Did you run all of your most recent changes locally to make sure everything is working?
